### PR TITLE
feat(caseking): add 3060ti/3070 cards / add missing 3080/3090 cards

### PR DESCRIPTION
**Pull request by wofnull**
### Description

Huge Update to Caseking.ts, added the currently missing 3060ti / 3070 cards, aswell as not yet added cards from the series 3080 / 3090

### Testing
Cards are checked so far and are working as expected.

### CODE REVIEW

**Comment by wofnull** [comment]
please checkup what happened here for the lint process: 
lookup.ts aswell as amazon.ts was not changed / touched with this change 
Only additions to caseking.ts aswell as store.ts ( new card types ) and filter.md ( new card / brand combinations )

**Comment by jef** [comment]
Fixing now!

File: src/store/model/store.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ export type Pricing = {
 
 export type Brand =
 	| 'test:brand'
-	| 'captcha-deterrent'
```
Comment (jef) [inline src/store/model/store.ts:15]: You'll need to put these back in order for linting to pass.

File: src/store/model/store.ts [hunk 2]
```diff
@@ -150,10 +149,11 @@ export type Model =
 	| 'vision'
 	| 'xbox series s'
 	| 'xbox series x'
-	| 'xc black'
 	| 'xc gaming'
+	| 'xc black'
```
Comment (jef) [inline src/store/model/store.ts:153]: Let's put this back in lexicographical order please :)

File: src/store/model/store.ts [hunk 3]
```diff
@@ -184,12 +184,6 @@ export type Labels = {
 	maxPrice?: Pricing;
 };
 
-export type CaptchaDeterrent = {
-	hardLinks?: string[];
-	searchUrl?: string;
-	searchTerms?: string[];
-};
-
```
Comment (jef) [inline src/store/model/store.ts:192]: This will need to go back.

Thanks!

**Review by jef — CHANGES_REQUESTED** [review]

**Comment by jef** [comment]
Fixed it for you in your branch. Thanks for the additions!

**Review by jef — APPROVED** [review]

**Pull request merged**